txclib/utils.py

```diff
@@ -467,8 +493,12 @@ def files_in_project(curpath):
             del dirs[:]
             continue
 
-        for f in files:
-            yield os.path.realpath(os.path.join(root, f))
+        for file_path in files:
+            full_path = os.path.realpath(os.path.join(root, file_path))
+            match = expression_regex.match(posix_path(full_path))
+            if match:
+                lang = match.group(1)
```

Comment:
You need to check for IndexError here and raise an exception just like it was happening before in https://github.com/transifex/transifex-client/pull/221/files#diff-17fee4a0664107b8fa4ca8129ec6a4f1L286

Reply:
@SofiaMargariti I removed it intentionally since we catch such cases when the user configures the expression [1], but this doesn't cover the case where a user changes the configuration file by hand. So you are right, I will re-add it.
[1]: https://github.com/transifex/transifex-client/blob/master/txclib/commands.py#L159